app/src/main/java/com/openclassrooms/realestatemanager/model/Estate.kt

```diff
@@ -0,0 +1,10 @@
+package com.openclassrooms.realestatemanager.model
+
+import java.util.*
+
+data class Estate(val id: Int, val name: String, val type: Type, val price: Double, val surface: Float, val numberOfRoom: Int, val description: String, val housseNumber: Int,val street: String, val ZIPCode: String,val city: String, val status: Int, val entryDate: Calendar, var saleDate: Calendar?, val agent: String) {
+
+    enum class Type {
+        House, Duplex, Flat, Penthouse
```

Comment:
Par convention, les enums sont EN_MAJUSCULE en Kotlin

Reply:
Ah non, apparemment c'est acceptable aussi :)

"For enum constants, it's OK to use either uppercase underscore-separated names ([screaming snake case](https://en.wikipedia.org/wiki/Snake_case)) (enum class Color { RED, GREEN }) or upper camel case names, depending on the usage."
https://kotlinlang.org/docs/coding-conventions.html#property-names

--> Ca me semble pratique si un jour tu changes l'enum en sealed class notamment.

Reply:
Ok